package/MDAnalysis/core/AtomGroup.py

```diff
@@ -606,6 +606,29 @@ def velocity(self, vals):
             raise NoDataError("Timestep does not contain velocities")
 
     @property
+    def occupancy(self):
+        """Access occupancy values.
+
+        If available can have a value between 0 and 1
+
+        :Returns: float
+
+        .. versionadded:: 0.12"""
+        try:
+            return self.universe.coord.data['occupancy'][self.index]
+        except KeyError:
+            raise NoDataError('Timestep does not contain occupancy')
+
+    @occupancy.setter
+    def occupancy(self, _occupancy):
+        """Set occupancy for atom"""
+        try:
+            self.universe.coord.data['occupancy'][self.index] = _occupancy
```

Comment:
Yeah `Timestep` arrays are the master arrays, always len(n_atoms), and index/indices are always global